1-exercises/B-todo-list/script.js

```diff
@@ -1,12 +1,31 @@
 function todoList(todos) {
-  // Write your code here...
+  todos.forEach((item) => {
+    let div = document.getElementById("content");
+    let ul = document.createElement("ul");
+    div.appendChild(ul);
+    let li = document.createElement("li");
+    ul.appendChild(li);
+    li.innerHTML = item.todo;
+    li.addEventListener("click", () => {
+      li.classList.toggle("onClick");
```

Comment:
See comment below, regarding toggle. best use to handle this scenario in this way. Well done.